Navigation.css

```diff
@@ -0,0 +1,82 @@
+body {
+    background-color: aliceblue;
+  }
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    height: 100vh;
+  }
+  .alert-container {
+    align-items: center;
+    background-color: #f0f0f0;
+    border-radius: 0.3rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: center;
+    padding: 2rem;
+    width: 50rem;
+  }
+  
+  .navbar {
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+    border-radius: 5px;
+  }
+  /* .navbar.navbar:second-child {
+    justify-self: center;
+  } */
+  .action-buttons {
+    margin-left: auto;
+    display: flex;
+    gap: 1.2rem;
+    margin-right: 1rem;
+    align-items: center;
+  }
+  .input-container {
+    margin-left: 6rem;
+    margin-right: 6rem;
+    padding: 0.3rem 5rem;
+    border-radius: 5px;
+  }
+  .logo-img {
+    background-image: url("./Images/CLogo.svg");
+    background-repeat: no-repeat;
+    mix-blend-mode: multiply;
+    width: 3rem;
+    height: 3rem;
+    background-size: 3rem;
+    background-color: transparent;
+    overflow: hidden;
+    margin-left: 1rem;
+  }
+  .dropdown {
+    position: relative;
+    display: inline-block;
+  }
+  /* Dropdown Content (Hidden by Default) */
+  .dropdown-content {
+    display: none;
+    position: absolute;
+    background-color: #f1f1f1;
+    min-width: 160px;
+    box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+    z-index: 1;
+    right: -500%;
```

Comment:
is it required this value to be -500%?

Reply:
I had to align it to middle of Profile, honestly, this is what worked. Can you please suggest a better way?

Reply:
-50% didn't work?